### PR TITLE
WIP Enhanced UI

### LINKED ISSUE #28: Enhance the UI
Modifications in order of priority **(first one is highest priority)**
- [x] Title
- [x] Post Templates
- [x] Forms
- [x] News Feed

### PR DESCRIPTION
**Pull request by kunal-kushwaha**
Fixes #28

### CODE REVIEW

**Comment by jessieAnhNguyen** [comment]
Enhanced the UI
- Changed the background color and font
- Changed the title and form

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ function Home() {
           <Transition.Group>
             {posts &&
               posts.map((post) => (
-                <Grid.Column key={post.id} style={{ marginBottom: 20, marginTop: 20 }}>
+                <Grid.Column class="post-container" key={post.id} style={{ marginBottom: 20, marginTop: 20 }}>
```
Comment (anikethsaha) [inline client/src/pages/Home.js:34]:
```suggestion
                <Grid.Column className="post-container" key={post.id} style={{ marginBottom: 20, marginTop: 20 }}>
```

**Comment by kunal-kushwaha** [comment]
LGTM

**Comment by anikethsaha** [comment]
@kunal-kushwaha  I had a pending request change in this. We did decide to wait for two approvals.

No issue,as we are running out of time,  just need to make that consistent from next time .  👍

**Comment by jessieAnhNguyen** [comment]
Yes sure I'll pay attention that next time @anikethsaha . Thanks for your comment!

**Pull request merged**